code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -294,7 +294,7 @@
 	icon_state = "smallbottle"
 	item_state = "bottle"
 	list_reagents = list(/datum/reagent/water = 49.5, /datum/reagent/fluorine = 0.5)//see desc, don't think about it too hard
-	custom_materials = list(/datum/material/glass=0)
+	custom_materials = list(/datum/material/plastic=1000)

```

Comment:
>1000

haha no

Reply:
No not 0 either

Reply:
What should it be, then? It takes a sheet to craft a small one and 3 sheets for a big one, I don't even know what this does